Sources/Datadog/RUM/RUMVitals/VitalMemoryReader.swift

```diff
@@ -0,0 +1,33 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+internal class VitalMemoryReader: VitalReader {
+    static let TASK_VM_INFO_COUNT = MemoryLayout<task_vm_info>.size / MemoryLayout<natural_t>.size
+
+    func readVitalData() -> Double? {
```

Comment:
The return value seems to be "used memory in bytes", so having `UInt64` would be a cleaner type for it, no? Also, I'm missing some comment either for this method or the entire class, that would explain which memory metric do we actually capture - it's quite hard to guess it from the low-level APIs below 🙂.

Reply:
Sounds fair 👍